openebs/pkg/volume/maya_api.go

```diff
@@ -200,9 +196,9 @@ func (v OpenEBSVolume) DeleteVolume(vname string, namespace string) error {
 
 	code := resp.StatusCode
 	if code != http.StatusOK {
-		glog.Errorf("HTTP Status error from maya-apiserver: %v\n", http.StatusText(code))
-		return err
+		return errors.New(http.StatusText(code))
```

Comment:
Please check if we can extract the original error message that is being sent from maya api server? Also please check if maya api server is able to send exact error message/reason along with err code. maya api server need not send generic error reasons.